package.json

```diff
@@ -122,6 +123,7 @@
     "pg-query-stream": "^3.2.0",
     "pluralize": "^8.0.0",
     "plyrue": "^2.1.4",
+    "poi": "^12.10.3",
```

Comment:
Please revert this change and install missing `mississipi` package (required in amazon storage file)